server/JsDbg.Gdb/JsDbg.py

```diff
@@ -248,7 +249,8 @@ def LookupSymbolName(pointer):
 def ReadMemoryBytes(pointer, size):
     inferior = gdb.selected_inferior()
     # Note: will throw an error if this includes unmapped/ unreadable memory
-    return inferior.read_memory(pointer, size).hex()
+    buf = inferior.read_memory(pointer, size)
+    return binascii.hexlify(bytearray(buf))

```

Comment:
Based on our discussions in slack, this might have unintended consequences since it adds some decorators to the string output.

Reply:
Please take another look - I fixed this with a version check